changelog/content/released/v0.14.0-resource-discovery.md

```diff
@@ -0,0 +1,11 @@
+---
+subtitle: "released on 2025-04-10"
+date: 2025-04-10T07:00:47+02:00
+weight: 1031
```

Comment:
```suggestion
weight: 1035
```

This needs to be incremented in 2, but looks like I forgot to publish 0.13 so doing 4